handler.go

```diff
@@ -51,6 +51,16 @@ func (m *methods) On(method string, f interface{}) error {
 
 	return nil
 }
+/**
+Remove corresponding message processing functions bound to supplied methods 
+ */
+func (m *methods) RemoveHandlers(methods []string) {
+	m.messageHandlersLock.Lock()
```

Comment:
may not be needed, an audit of how messageHandlers is used is needed to confirm though...